cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-thanos-object-storage/externalsecret.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+namespace: open-cluster-management-observability
+metadata:
+  name: open-cluster-management-observability-thanos-object-storage
+  namespace: open-cluster-management-observability
+spec:
+  refreshInterval: 15s
+  secretStoreRef:
+    name: nerc-cluster-secrets
+    kind: ClusterSecretStore
+  target:
+    name: thanos-object-storage
+    template:
+      type: Opaque
+      engineVersion: v2
+      metadata:
+        namespace: open-cluster-management-observability
+      data:
+        thanos.yaml: |
+          type: s3
+          config:
+            bucket: "{{ .data.bucket }}"
+            bucketnames: "{{ .data.bucket }}"
+            endpoint: "s3.openshift-storage.svc:443"
+            insecure: false
+            access_key: "{{ .data.access_key }}"
+            secret_key: "{{ .data.secret_key }}"
+            http_config:
+              tls_config:
+                ca_file: "/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+  data:
+  - secretKey: bucket
+    remoteRef:
+      key: $ENV/$CLUSTER/open-cluster-management-observability-thanos-object-storage
+      property: bucket
+  - secretKey: access_key
+    remoteRef:
+      key: $ENV/$CLUSTER/open-cluster-management-observability-thanos-object-storage
+      property: access_key
+  - secretKey: secret_key
+    remoteRef:
+      key: $ENV/$CLUSTER/open-cluster-management-observability-thanos-object-storage
```

Comment:
the 3 keys will also need to be patched in the overlay

Reply:
Thanks @naved001

Reply:
Thanks @naved001 @hpdempsey  I have patched these values for nerc-ocp-prod in the latest commits.